x/feeabstraction/ante/cosmos/fee_test.go

```diff
@@ -53,6 +53,10 @@ func TestDeductFeeDecorator(t *testing.T) {
 	// Create the funder account
 	app.AccountKeeper.SetAccount(ctx, app.AccountKeeper.NewAccountWithAddress(ctx, founder))
 
+	// Reactivate keeper since it was disabled on abci
+	err := app.FeeAbstractionKeeper.Params.Set(ctx, types.DefaultParams())
+	require.NoError(t, err)
```

Comment:
This is weird, on `SetupWithContext` we should be setting the module with defaults. So why re-set here?